.github/workflows/main.yml

```diff
@@ -440,18 +440,18 @@ jobs:
             $null = New-Item -ItemType Directory -Force -Path install_temp
           }
 
-          Expand-Archive -Path "$(Get-ChildItem -filter "OBS-Studio-*-Win-x86.zip" -File)" -DestinationPath install_temp
-          Expand-Archive -Path "$(Get-ChildItem -filter "OBS-Studio-*-Win-x64.zip" -File)" -Force -DestinationPath install_temp
+          Expand-Archive -Path "$(Get-ChildItem -filter "obs-studio-windows-*-x86.zip" -File)" -DestinationPath install_temp
+          Expand-Archive -Path "$(Get-ChildItem -filter "obs-studio-windows-*-x64.zip" -File)" -Force -DestinationPath install_temp
 
           CI/windows/03_package_obs.ps1 -CombinedArchs -Package
 
-          $ArtifactName = (Get-ChildItem -filter "OBS-Studio-*-Win-x86+x64.zip" -File).Name
+          $ArtifactName = (Get-ChildItem -filter "obs-studio-windows-*-x86+x64.zip" -File).Name
```

Comment:
Same as above.

This line needs to be changed to:
```
$ArtifactName = Get-ChildItem -filter "obs-studio-*-windows-${{ matrix.arch }}.zip" -File
```

The pattern here comes from `DestinationPath` in `CI/windows/03_package_obs.ps1`:
```
    $FileName = "obs-studio-${VersionString}-windows"
```
```
            DestinationPath = "${FileName}-x86+x64.zip"
```
So `DestinationPath` is: `obs-studio-${VersionString}-windows-x86+x64.zip`.

Reply:
Fixed